test/persistence.test.js

```diff
@@ -126,5 +126,39 @@ describe('given a connected persistence', function() {
       });
     });
 
+    it('should set/get a single standalone key with TTL', function(done) {
+      var key = 'persistence.messages.object.test';
+      var objectValue = {
+        foo: 'bar'
+      };
+
+      var keyTTL = 2;
+      Persistence.persistKey(key, objectValue, keyTTL);
+      Persistence.readKey(key, function (reply) {
+        if (reply) {
+          assert.deepEqual({ foo: 'bar' }, reply);
+          done();
+        }
```

Comment:
These tests should also assert that there is a ttl set. Persistence.redis().ttl(key, callback) should get you that. It should have a ttl between 0 and 2. (based on time expired)